data/json/items/ammo/300BLK.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "id": "300BLK_supersonic",
+    "type": "AMMO",
+    "name": ".300 AAC Blackout",
+    "description": ".300 AAC Blackout is an intermediate cartridge that achieves ballistics similar to the 7.62x39 but allows use on the AR-15 platform.  The round is necked-up from the 5.56mm NATO, but feeds from a STANAG magazine.  It requires a specific barrel, so will not work in a standard M4 or similar carbine.  This loading is supersonic.",
+    "weight": 18,
+    "volume": 1,
+    "price": 3500,
+    "material": [ "brass", "powder" ],
+    "symbol": "=",
+    "color": "yellow",
+    "count": 30,
+    "stack_size": 47,
```

Comment:
Stack sizes for ammunition are typically the number you'd get in an average commercial box. Why 47 here?

Reply:
Count is the number per box, stack size is the number per volume of space. 47 is the same size as .223/5.56. 30 is a weird number; ammunition is usually sold in boxes of 20, 50, 100, 500, or 1,000.

Reply:
Yeah, I literally copied what it was for the .223. Want it adjusted?

Reply:
No leave it as is. I was mistaken about what `stack_size` is for.